protostar/cairo/cairo_bindings.py

```diff
@@ -12,11 +12,13 @@ class TestCollectorOutput:
     test_names: list[str]
 
 
-def compile_starknet_contract(
+def compile_starknet_contract_from_path(
     input_path: Path,
     output_path: Optional[Path] = None,
     cairo_path: Optional[list[Path]] = None,
 ) -> Optional[str]:
+    if output_path:
+        Path(output_path).parent.mkdir(parents=True, exist_ok=True)
```

Comment:
[nitpick]: can this be extracted to a function? `ensure_output_path` would be an okay name

Reply:
[add ensure_output_path](https://github.com/software-mansion/protostar/pull/1551/commits/f9d1dcc794a4ffe3136b8e16da1dc39ef03611d5)